lib/Optimizer/GraphOptimizer/GraphOptimizer.cpp

```diff
@@ -1072,13 +1073,15 @@ bool MergeBatchedAdd::run(Function *F) {
     // We found a sequence of batched-add-slice that cover the input tensor.
     // We can transform the graph and create one big batched-add.
     std::vector<Node *> newSlices;
+    assert(order.size() > 1 && "order must contain at least 2 SliceNodes.");
```

Comment:
nit: I think current preference is GLOG instead of old type asserts.

Reply:
Good call. We haven't updated GraphOptimizer yet though to use glog. Let's make all of the changes there at once.

Reply:
sounds good.